Frameworks/UIKit/UIApplication.mm

```diff
@@ -698,7 +698,8 @@ - (void)setApplicationIconBadgeNumber:(int)num {
     WUNBadgeNotification* notification = [WUNBadgeNotification makeBadgeNotification:doc];
     WUNBadgeUpdater* updater = [WUNBadgeUpdateManager createBadgeUpdaterForApplication];
 
-    [updater update:notification];
+    // TODO #: 0x803e0208 : The notification platform does not have the proper privileges to complete the request.
```

Comment:
#1201

---
In reply to: [83327322](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r83327322) [](ancestors = 83327322)